scripts/zones/Behemoths_Dominion/mobs/Behemoth.lua

```diff
@@ -15,6 +15,16 @@ function onMobInitialize(mob)
 end;
 
 -----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    if (LandKingSystem_NQ > 0 or LandKingSystem_HQ > 0) then
+        GetNPCByID(17297459):setStatus(STATUS_DISAPPEAR);
```

Comment:
I wasn't sure either, but thought since there are 2 different options in the settings.lua I would just separate them. But can change it anyway. The only "important" part is not to spawn the mobs when the settings.lua says they should be force popped